cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeaBasedPluginInstaller.java

```diff
@@ -47,23 +45,14 @@ private String readBuildVersion() {
     Path buildFile = commandlet.getToolPath().resolve(BUILD_FILE);
     if (context.getSystemInfo().isMac()) {
       MacOsHelper macOsHelper = new MacOsHelper(context);
-      Path rootToolPath = macOsHelper.findRootToolPath(this.commandlet, context);
-      buildFile = rootToolPath.resolve("IntelliJ IDEA" + generateMacEditionString() + ".app").resolve("Contents/Resources").resolve(BUILD_FILE);
+      Path appPath = macOsHelper.findAppDir(macOsHelper.findRootToolPath(this.commandlet, context));
```

Comment:
Nice that we now do not hardcode the app name here anymore.
However, IMHO the method `findRootToolPath` was already buggy before and does not honor the edition (always uses tool name also as edition what is incorrect here).
Also `"default"` is hardcoded there as redundant string literal instead of reusing `ToolRepository.ID_DEFAULT` so could you also please improve that alongside?